tarantool/connection.py

```diff
@@ -804,6 +814,18 @@ def __init__(self, host, port,
         """
         # pylint: disable=too-many-arguments,too-many-locals,too-many-statements
 
+        if host is None and port is None and socket_fd is None:
+            raise ConfigurationError("need to specify both host and port, "
+                                     "port only in the case on Unix sockets, "
+                                     "or socket_fd in the case of socket fd")
```

Comment:
```suggestion
raise ConfigurationError("need to specify host/port, "
                                     "port (in case of Unix sockets)"
                                     "or socket_fd")
```